tool/doc.dart

```diff
@@ -245,6 +246,16 @@ String getBadges(String rule, [String? fixStatus]) {
   return sb.toString();
 }
 
+String getFixStatus(LintRule rule, Map<String, String> fixStatusMap) {
+  for (var code in rule.lintCodes) {
+    var status = fixStatusMap[code.uniqueName.substring(9)];
+    if (status != null) {
+      return status;
```

Comment:
If there are multiple codes, and the first has a status of 'needsFix', but the second has a status of 'hasFix', which should we show? What if the order is reversed?

Reply:
Hrmmm.  Good point.  I guess we probably want to emphasize the positive here 😉 

Probably best to look for a `hasFix` in any order.  I'll update.